packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -0,0 +1,3 @@
+from pkgpanda.util import load_json
+
+dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
Note: there is no conflict downstream as these files will live in totally different namespaces, but I agree with changing this to expanded_config and will get that in one of the future cleanups